.spellcheck-en-custom.txt

```diff
@@ -108,9 +112,11 @@ Korol
 Kruse
 Kubernetes
 Kunstle
+Labradorite
 LGTM
 LGTMs
 lhawthorn
+licenced
```

Comment:
I don't think this is a word.

According to the internet:
> If you're American, use "license."
> If you're British, use "licence" for the noun (i.e., papers, a permit) but "license" for the verb (i.e., to allow, to give permission).
> UK example: I have a licence. I am licensed to drive.

I try to be accepting of non-American spelling but not only does it appear to be used as a verb, but in this case sticking to the American way is just much simpler.